### ISSUE TITLE
New BSIP: Free TRANSFERs with POS 

### ISSUE THREAD

**Issue by froooze** [bitshares/bsips#212]
# Abstract
Newer blockchains (EOS, IOTA, NANO ... ) want to improve adoption and offer new use cases with zero transfer fees. There are different concepts to prevent spam attacks, but POS looks like the most convenient one.

# Implementation
Locking up `x` BTS allows `x`/`BTS for free transfer` during `time duration`. One staking account can provide free transfers for other accounts with a whitelist function. 

## Variables
- `time duration` 
- `BTS for free transfer` 
- Whitelist

## Question
- Do margin positions also count as locked BTS?

**Comment by froooze** [comment]
Free transfer feature requested:
https://bitsharestalk.org/index.php?topic=22263.msg335931#msg335931

**Comment by pmconrad** [comment]
> Do margin positions also count as locked BTS?

The BTS collateral in margin positions is not locked up - a variable part of it is backing the debt and can be taken out at any time due to margin calls and/or forced settlement.

> A margin position quote for free transfers ?

I see no justification for this.

> Locking up x BTS allows x/BTS_per_FT during period_for_FT.

You need to specify what "Locking up" means.

Also, using explicit parameters has the downside that these parameters will need to be discussed and defined explicitly. I would prefer an automatic approach, similar to the rate-limited bandwidth model used by STEEM.

> POS wit coin-days

There are other models. IMO simple coin-days is not good because it allows accumulating over long time followed by heavy spamming in a short time.

**Comment by froooze** [comment]
@pmconrad 
I agree on your answers and coin-days was the wrong wording for my concept. 

I look for a concept which needs period_for_FT to reload balance from 0% to 100%
